proto/accessgraph/v1alpha/aws.proto

```diff
@@ -584,3 +586,41 @@ message AWSOIDCProviderV1 {
   // last_sync_time is the time when the resource was last synced.
   google.protobuf.Timestamp last_sync_time = 8;
 }
+
+// AWSKMSKeyV1 defines the KMS key details.
+message AWSKMSKeyV1 {
+  // arn is the key ARN.
+  string arn = 1;
+  // created_at is the time when the key was created.
+  google.protobuf.Timestamp created_at = 2;
+  // tags is the list of tags that are attached to the key.
+  repeated AWSTag tags = 3;
+  // account_id is the ID of the AWS account that the key belongs to.
+  string account_id = 4;
+  // last_sync_time is the time when the resource was last synced.
+  google.protobuf.Timestamp last_sync_time = 5;
+  // aliases is the list of aliases that are attached to the key.
+  repeated string aliases = 6;
+  // policy_document is the JSON document that defines the policy.
+  bytes policy_document = 7;
+  // region is the AWS region that the key belongs to.
+  string region = 8;
+  // multi_region_key_type is the type of the multi-region key.
+  MultiRegionKeyType multi_region_key_type = 9;
+  // hsm_cluster_id is the ID of the HSM cluster that the key belongs to, if any.
+  string hsm_cluster_id = 10;
+}
+
+// MultiRegionKeyType is the type of a multi-region key type (primary, replica,
+// none for single region).
+enum MultiRegionKeyType {
+  // MULTI_REGION_KEY_TYPE_UNSPECIFIED is a unspecified multi region key type
+  MULTI_REGION_KEY_TYPE_UNSPECIFIED = 0;
+  // MULTI_REGION_KEY_TYPE_NONE is not a multi region key type but a local or
+  // single region key.
+  MULTI_REGION_KEY_TYPE_NONE = 1;
+  // MULTI_REGION_KEY_TYPE_PRIMARY is a primary multi region key type
+  MULTI_REGION_KEY_TYPE_PRIMARY = 2;
+  // MULTI_REGION_KEY_TYPE_REPLICA is a replica multi region key type
+  MULTI_REGION_KEY_TYPE_REPLICA = 3;
+}
```

Comment:
Like mentioned above, I'd propose we simplify this and just use a string instead of an enum.